crates/noirc_driver/src/lib.rs

```diff
@@ -155,10 +150,13 @@ pub fn compile_main(
     let main = match context.get_main_function(&crate_id) {
         Some(m) => m,
         None => {
+            // TODO: This error might be a better to exist in Nargo
```

Comment:
Resolve TODO

Reply:
This will be easier when the refactor for #2075 happens so I'll open an issue for it.

Reply:
```suggestion
            // TODO(#2155): This error might be a better to exist in Nargo
```